lib/ui/src/modules/ui/components/stories_panel/sub_header.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { baseFonts } from '@storybook/components';
+
+const wrapperStyle = {
+  marginTop: 20,
+};
+
+const headingStyle = {
+  ...baseFonts,
+  textTransform: 'uppercase',
+  // letterSpacing: '1.5px',
```

Comment:
Looks like it's worth uncommenting. Uppercase text is more readable when it has some letter-spacing

Reply:
Changed it to letterSpacing: '1.2px'